tests/by-util/test_mcookie.rs

```diff
@@ -63,3 +63,34 @@ fn test_seed_files_and_max_size() {
         file2.path().to_str().unwrap()
     ));
 }
+
+#[test]
+#[cfg(unix)] // Character devices like /dev/zero are a Unix concept
+fn test_char_device_input() {
+    let res_no_limit = new_ucmd!().arg("-f").arg("/dev/zero").succeeds();
+
+    let stdout_no_limit = res_no_limit.no_stderr().stdout_str();
```

Comment:
A detail: you could call `trim_end()` on this line, which allows you to remove it from lines 73 and 75. The same applies for the `stdout_verbose` var.

Reply:
Feedback received, updating shortly.